packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -86,21 +94,40 @@ class ToOOXMLVisitor {
                     let textValueTag = TEXT_RULE(subNode.text);
 
                     let tag = TEXT_WRAPPER_RULE(propertyTag, textValueTag);
+
+                    if (isLinkPropertyPresent) {
+                        // some rels are already present
+                        // To avoid overwrite ids start our
+                        // rels from 10
+                        let relationShipId = 'rId' + (this.relationShips.length + 10).toString();
+                        tag = LINK_RULE(tag, relationShipId);
+                    }
                     this.tags = [...this.tags, tag];
                 } else if (this.getClass(subNode) === TRANSFORMED_NODES.code) {
                     let propertyTag = CODE_PROPERTIES_RULE();
+                    let isLinkPropertyPresent = false;
                     for (let property of properties) {
                         if (property === TRANSFORMED_NODES.emphasize) {
                             propertyTag += EMPHASIS_RULE();
                         } else if (property === TRANSFORMED_NODES.strong) {
                             propertyTag += STRONG_RULE();
+                        } else if (property === TRANSFORMED_NODES.link) {
+                            isLinkPropertyPresent = true;
+                            propertyTag+= LINK_PROPERTY_RULE();
                         }
                     }
                     propertyTag = TEXT_STYLES_RULE(propertyTag);
 
                     let textValueTag = TEXT_RULE(subNode.text);
 
                     let tag = TEXT_WRAPPER_RULE(propertyTag, textValueTag);
+                    if (isLinkPropertyPresent) {
+                        // some rels are already present
+                        // To avoid overwrite ids start our
+                        // rels from 10
```

Comment:
What do you mean by this? Could you elaborate?

Reply:
@algomaster99 
If you see the below:
```xml
<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
          <Relationship Id="rId1" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/numbering" Target="numbering.xml"/>
          <Relationship Id="rId2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/styles" Target="styles.xml"/>

        </Relationships>
```

Here some rels are already used basically for styling, themes, or settings, etc.
Hence, to avoid conflict I start with 10.
Actually, it would be `11`. I will correct this typo